README.md

```diff
@@ -1,6 +1,8 @@
 <h1><img src="http://enroute.osgi.org/img/enroute-logo-64.png" witdh=40px style="float:left;margin: 0 1em 1em 0;width:40px">
 OSGi enRoute</h1>
 
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io#https://github.com/Sandared/jmh-benchmarks/blob/master/benchmarks/src/main/java/io/jatoms/jmh/OSGiBenchmark2.java)
```

Comment:
1) Gitpod is a for-profit company. I am not sure why OSGi would want to promote them.

2) The referenced URL goes to your github account and I don't see that OSGi would want to link to that.

Reply:
1) True, but just to onboard users quickly and letting them play around wih the quickstarter app the 10 free hours are more than sufficient I think. I personally would love to just click a button and see the examples working and being able to play around with the code, but if thats interfering with OSGi-Alliance guidelines then of course it's a no-go. Didn't think about that before.
2) Sorry, Copy/Paste error. Will fix that.

Reply:
Re: 1) Gitpod is now free for open-source projects (see https://www.gitpod.io).